Sources/PodToBUILD/SwiftLibrary.swift

```diff
@@ -150,7 +150,23 @@ public struct SwiftLibrary: BazelTarget {
             }
         }
  
-        self.copts = AttrSet.empty
+        let swiftFlags = XCConfigTransformer.defaultTransformer(
+            externalName: externalName, sourceType: .swift)
+            .compilerFlags(for: fallbackSpec)
+
+        let objcFlags = XCConfigTransformer.defaultTransformer(
+            externalName: externalName, sourceType: .objc)
+            .compilerFlags(for: fallbackSpec)
+
+        let includes = objcFlags.filter { $0.hasPrefix("-I") }
+
+        // Insert the clang import flags
+        let clangImporterCopts = (includes + ["-DCOCOAPODS=1"])
+            .reduce([String]()) { $0 + ["-Xcc", $1] }
+        self.copts = AttrSet(basic: [
+            "-DCOCOAPODS",
+        ] + swiftFlags + clangImporterCopts)
```

Comment:
This adds both `-DCOCOAPODS=1` and `-DCOCOAPODS`?

Reply:
This adds `-DCOCOAPODS` and `-DCOCOAPODS=1` to the clang importer ( via `-Xcc` ) - by doing this, clang sees `-DCOCOAPODS=1`. I'm not 100% sure why it doesn't pass `-DCOCOAPODS` to swift, but this is the behavior in Xcode so I wanted to make it identical